### PR TITLE
Add multiclusterhub-repo WIP

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>
https://github.com/open-cluster-management/backlog/issues/9785

### CODE REVIEW

**Comment by zkayyali812** [comment]
/assign @qiujian16
Can you please review when you get a moment?

File: content/concepts/architecture.md [hunk 1]
```diff
@@ -45,3 +45,7 @@ The _Application console_ defines the user interface used to create and manage a
 ## Governance and risk (work in progress)
 
 _Governance and risk_ is the term used to define the processes that are used to manage security and compliance from the hub cluster. Ensure the security of your cluster with the extensible policy framework. After you configure a hub cluster and a managed cluster, you can create, modify and delete policies on the hub and apply policies to managed clusters.
+
+## Multiclusterhub Repo (work in progress)
```
Comment (qiujian16) [inline content/concepts/architecture.md:49]: i would have a Multiclusterhub installer section and Multiclusterhub Repo as a subsection of it.

Comment (nathanweatherly) [reply content/concepts/architecture.md:49]: @qiujian16 That's a good plan-- One section for the "Hub Operator" and the repo & other sub-components can be sub sections. 

We have a work item for adding the MCH Operator to the website here: https://app.zenhub.com/workspaces/applifecycle-issue-tracker-5e480a1f17e47394a67447b9/issues/open-cluster-management/backlog/9780

I'm starting on this work today and will complete it in Sprint 2. 

Can we merge in the MCH Repo section as-is to close our issue (and prevent code rot) and then I will add a note to the MCH Operator's issue to re-arrange?

Comment (qiujian16) [reply content/concepts/architecture.md:49]: I think you could have the section of "Hub Operator (work in progress)"  with only this "Multiclusterhub Repo (work in progress)" as a subsection, and leave other content as empty.

Comment (zkayyali812) [reply content/concepts/architecture.md]: Updated in most recent commit

**Comment by zkayyali812** [comment]
merge conflicts fixed

**Pull request merged**